Sources/MessagingPush/PushHandling/iOSPushEventListener.swift

```diff
@@ -4,7 +4,7 @@ import Foundation
 
 @available(iOSApplicationExtension, unavailable)
 // sourcery: InjectRegister = "PushEventHandler"
-class iOSPushEventListener: PushEventHandler {
+class IOSPushEventListener: PushEventHandler {
```

Comment:
There is a lint error requiring that all classes start with an uppercase letter. 

If this class was public-facing I may suggest we disable this rule. But being internal, not the biggest deal to use `IOS` 🤷‍♀️

Reply:
Apart from the linter, maybe the prefix iOS isn't needed after all. I mean, it would make sense if we have IOSPushEvenListner, MacOSEventListner, etc. But we don't.